logger.go

```diff
@@ -32,6 +32,8 @@ type Logger struct {
 	mu MutexWrap
 	// Reusable empty entry
 	entryPool sync.Pool
+	// Function to exit the application, defaults to `Exit()`
+	Exit exitFunc
```

Comment:
The symbol Exit should rather be a method of the logger object to be backward compatible. I think users may also want to have their registered handlers to be called also, therefore we want to istead to make configurable the os.Exit call from logrus.Exit